python/tk_multi_loader/dialog.py

```diff
@@ -1276,7 +1278,9 @@ def _on_search_text_changed(self, pattern, tree_view, proxy_model):
         proxy_model.setFilterFixedString(pattern)
 
         # change UI decorations based on new pattern.
-        if pattern and len(pattern) > 0:
+        # for performance, make sure filtering only kicks in
+        # once we have typed at least one character
+        if pattern and len(pattern) > constants.TREE_SEARCH_TRIGGER_LENGTH:
```

Comment:
Suggest making the value of `TREE_SEARCH_TRIGGER_LENGTH` be `2` and making this `>=` just for clarity. i.e. the tree search isn't triggered until the length is 2 or greater.